src/storage-preview/azext_storage_preview/tests/latest/test_azcopy_scenarios.py

```diff
@@ -76,3 +75,66 @@ def test_azcopy_flow(self, resource_group, storage_account_info, test_dir):
 
         self.cmd('storage azcopy blob delete -c {} --account-name {} --recursive'.format(
             container, storage_account))
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    @StorageTestFilesPreparer()
+    def test_azcopy_sync(self, resource_group, storage_account_info, test_dir):
```

Comment:
So where are the recordings for these new tests? I don't see any in the PR.

Reply:
They are same as the previous test test_azcopy_flow.

Reply:
It would be good to re-record them and commit the updated recordings.

Reply:
This is live test. The container name is not same if reruning the test as ScenarioTest.